hippunfold/workflow/rules/common.smk

```diff
@@ -132,17 +132,18 @@ def get_final_qc():
     qc = []
     #right now can only do qc from cropT1w space 
     if 'cropT1w' in config['output_spaces']:
-        qc.extend(
-            expand(
-                bids(
-                        root='results',
-                        datatype='qc',
-                        suffix='regqc.png',
-                        from_='subject', 
-                        to=config['template'],
-                        **config['subj_wildcards']),
-                    allow_missing=True)
-            )
+
+    #    qc.extend(
```

Comment:
not sure whether this is intended to remain commented out - I think this QC issue was already fixed?

Reply:
ah yes I think we can uncomment this, I'll make that change